Samples/WinMLSamplesGallery/WinMLSamplesGallery/Common/Prediction.cs

```diff
@@ -0,0 +1,12 @@
+﻿// To learn more about WinUI, the WinUI project structure,
+// and more about our project templates, see: http://aka.ms/winui-project-info.
+
+namespace WinMLSamplesGallery.Controls
+{
+    public class Prediction

```

Comment:
why is the "Prediction" class inside the "Controls" namespace? I would assume controls is about the UI, but the prediction object is not limited to UI usage, right?

Reply:
It is only used for showing the UI in the ImageClassification sample